modules/ROOT/partials/deployment/nginx_full_example.adoc

```diff
@@ -28,7 +29,7 @@ server {
         ssl_dhparam /etc/letsencrypt/ssl-dhparams.pem;
 
         location / {
-            proxy_pass http://localhost:{ocis_port};
+            proxy_pass http://127.0.0.1:{ocis_port};
```

Comment:
hmmm - do we need to put a literal IP local-host address here?
What happens to IPv6 in this case? It's "localhost" equivalent address won't be covered?

Reply:
ah, you are right, will change

Reply:
Reverted